plutus-tx-plugin/src/PlutusTx/Compiler/Builtins.hs

```diff
@@ -452,6 +455,9 @@ defineBuiltinTerms = do
             PLC.Bls12_381_mulMlResult -> defineBuiltinInl 'Builtins.bls12_381_mulMlResult
             PLC.Bls12_381_finalVerify -> defineBuiltinInl 'Builtins.bls12_381_finalVerify
 
+    -- Bitwise operations
+    defineBuiltinTerm annMayInline 'Builtins.integerToByteString               $ mkBuiltin PLC.IntegerToByteString
+    defineBuiltinTerm annMayInline 'Builtins.byteStringToInteger               $ mkBuiltin PLC.ByteStringToInteger
```

Comment:
Should be moved under the pattern matching right above, following the pattern.

Reply:
Thanks.  I'm not sure how that ended up there.